### PR TITLE
fix: kickOut does not throw itself

### LINKED ISSUE #1662: KickOut Should Not Throw
**What is the Problem Being Solved?**

Currently, we cannot `kickOut` multiple seats in one turn because `kickOut` throws. Instead, it should return an error which we can throw if we wish. 

**Description of the Design**

`zcfSeat.kickOut` should return an error, and should not itself throw.

**Security Considerations**

We should make sure nothing depended on this functionality.

**Test Plan**
Current tests, also, perhaps adding a test in which multiple seats get kicked out in one turn.

### PR DESCRIPTION
**Pull request by katelynsills**
Closes #1662

### CODE REVIEW

File: packages/zoe/test/unitTests/contracts/test-zcf.js [hunk 1]
```diff
@@ -27,7 +28,43 @@ test('zoe - test zcf', async t => {
...
+  const userSeat2 = await E(zoe).offer(invitation2);
+  const userSeat1Result = await E(userSeat1).getOfferResult();
+
+  t.is(userSeat1Result, 'ok');
+
+  await E(userSeat2).getPayouts();
+  // Results in "Unhandled rejection"
+  // E(userSeat2).getOfferResult();
+
+  await t.throwsAsync(() => E(userSeat2).getOfferResult());
```
Comment (katelynsills) [inline packages/zoe/test/unitTests/contracts/test-zcf.js]: This currently isn't being run, but I can't figure out why. I get "Planned for 1 assertion, but got 3."

**Comment by katelynsills** [comment]
@Chris-Hibbert  test added

**Review by michaelfig — APPROVED** [review]
Drive-by review.  LGTM!

The automatic throw was something I didn't expect when using `kickOut`, so I'm glad to see it gone.

**Review by Chris-Hibbert — APPROVED** [review]

**Review by erights — APPROVED** [review]
LGTM. Definitely better.

**Pull request merged**